presto-parser/src/main/java/io/prestosql/sql/tree/CreateMaterializedView.java

```diff
@@ -0,0 +1,146 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.prestosql.sql.tree;
+
+import com.google.common.collect.ImmutableList;
+
+import java.util.List;
+import java.util.Objects;
+import java.util.Optional;
+
+import static com.google.common.base.MoreObjects.toStringHelper;
+import static java.util.Objects.requireNonNull;
+
+public class CreateMaterializedView
+        extends Statement
+{
+    public enum Security
+    {
+        INVOKER, DEFINER
+    }
+
+    private final QualifiedName name;
+    private final Query query;
+    private final boolean replace;
+    private final boolean notExists;
+    private final List<Property> properties;
+    private final Optional<String> comment;
+    private final Optional<String> originalSql;
```

Comment:
Why is this being stored in textual form in this class?

Reply:
Sorry, neither needed nor should it be here. Removed.